server/src/main/java/org/elasticsearch/action/support/broadcast/node/TransportBroadcastByNodeAction.java

```diff
@@ -70,11 +71,13 @@
  * @param <Request>              the underlying client request
  * @param <Response>             the response to the client request
  * @param <ShardOperationResult> per-shard operation results
+ * @param <NodeContext>          the optional node context created by createNodeContext and passed to shardOperation
```

Comment:
nit (mainly adding cross-links)

```suggestion
 * @param <NodeContext>          an (optional) node context created by {@link #createNodeContext} on each node and passed to each call
 *                               to {@link #shardOperation}
```